content/en/docs/setup/quickstart.md

```diff
@@ -12,8 +12,8 @@ If you want to install Spinnaker on Lightweight Kubernetes (K3s) for proofs of c
 
 Options for installing Spinnaker in Kubernetes:
 
+- [Spinnaker Helm Chart](https://github.com/OpsMx/spinnaker-helm) is an open source helm chart to install/configure spinnaker.
 - [Spinnaker Operator for Kubernetes](https://github.com/armory/spinnaker-operator) is an open source Kubernetes Operator for deploying and managing Spinnaker. You can install a basic version of Spinnaker or use Kustomize files for advanced configuration.
-- [Spinnaker Operator](https://operatorhub.io/operator/spinnaker-operator) is an open source Helm chart for installing Spinnaker.
```

Comment:
I don't have much context here.  Perhaps @rebala can comment?  Does anyone know how to remove the operatorhub.io link if this really isn't supported?

It's probably not worth blocking this PR, but, it seems like we'd be better off if the repos we reference here were in github.com/spinnaker.  cc: @link108 .

Reply:
@dbyron-sf the operator link comes from https://github.com/k8s-operatorhub/community-operators/tree/main/operators/opsmx-spinnaker-operator. Need to raise a PR to remove the link from the operatorhub.io. 
@keerthibingi please remove the link from the operatorhub for the Spinnaker operator as well.

Reply:
Removed operator link as per your comments. Here is the PR raised [](https://github.com/k8s-operatorhub/community-operators/pull/2694)

Reply:
PR got merged into https://github.com/k8s-operatorhub/community-operators/tree/main/operators 
@dbyron-sf could you please review and approve this.

Reply:
Still would love to hear from @link108 about repos outside of the spinnaker github org, but generally this LGTM.